### PR TITLE
Homework6: Gleb Salnikov

### PR DESCRIPTION
**Pull request by glebsalnikov**

### CODE REVIEW

File: Task_06/ex1.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    # gzipped json file
+    gz_url = 'https://wiki.mozilla.org/images/f/ff/Example.json.gz'
+    gz_get = requests.get(gz_url)
+    with gzip.open(io.BytesIO(gz_get.content), 'r') as data:
+        file = json.loads(data.read())
+    print(file)
+
+if not arg.type:
+    # zip file
+    exists = os.path.isfile('FL_insurance_sample.csv')
```
Comment (belovmd) [inline Task_06/ex1.py]: it's code dublication. We can move this code to functions to reuse it.

**Comment by belovmd** [comment]
Good job but I don't see data serialization here. 
Here is our 2nd task condition:
Modify application so it can store data in 4 different formats. JSON, CSV, XML, YAML (use argparse)
--format json | xml | csv | yaml

**Comment by belovmd** [comment]
Also u can implement --download args.

**Comment by glebsalnikov** [comment]
> Also u can implement --download args.

But I did implement --download parsing, though it seems like --type. Or you mean my implementation isn't actually correct? I ask as you've attached (copy + paste apparently) such a comment not only to my pull request.

**Pull request closed without merging**